app/assets/javascripts/discourse/app/widgets/sidebar-toggle.js

```diff
@@ -5,17 +5,26 @@ export default createWidget("sidebar-toggle", {
 
   html() {
     const attrs = this.attrs;
+    this.appEvents.on(
```

Comment:
Do we remove this registration when the toggle is removed, if it is even removed?